webui/react/src/stores/users.tsx

```diff
@@ -55,12 +57,14 @@ export const useFetchUsers = (canceler: AbortController): (() => Promise<void>)
   const { updateUsers } = context;
 
   return useCallback(
-    async (cfg?: FetchUsersConfig): Promise<void> => {
+    async (cfg?: FetchUsersConfig): Promise<DetailedUserList | void> => {
       try {
         const config = cfg ?? {};
         const response = await getUsers(config, { signal: canceler.signal });
 
         updateUsers(() => Loaded(response.users));
+
+        return response;
       } catch (e) {
         handleError(e);
```

Comment:
`try, catch` is used in https://github.com/determined-ai/determined/pull/5603/files#diff-43faae95c76ff40b586c233c4c1545b430a7e64abb80d04d5bbbbd7e09be03f8R125,, so this try catch isnt much helpful.